functions/getMockedEventsSet.js

```diff
@@ -407,7 +407,9 @@ const getMockedEventsSet = () => {
     suggestedEventsWithOneProduction,
   ];
   const index = Math.floor(Math.random() * events.length);
-  return events[index];
+  const similarity = Math.random();
```

Comment:
I know this is just dummy bet shouldn't it be a percentage like `Math.floor((Math.random() * 100) + 1)`